ios/Sources/BackgroundGeolocationPlugin/BackgroundGeolocation.swift

```diff
@@ -183,7 +204,7 @@ public class BackgroundGeolocation : CAPPlugin, CLLocationManagerDelegate {
                     if clErr.code == .locationUnknown {
                         // This error is sometimes sent by the manager if
                         // it cannot get a fix immediately.
-                        return
+                return
```

Comment:
Likewise this indentation. Is there something weird going on with tabs?

Reply:
Fixed. (probably is my IDE auto format and change the indentation)